api/v1beta1/keystoneapi_types.go

```diff
@@ -89,6 +89,16 @@ type KeystoneAPISpec struct {
 	// Secret containing OpenStack password information for keystone KeystoneDatabasePassword, AdminPassword
 	Secret string `json:"secret"`
 
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=""
+	// TrustFlushArgs - Arguments added to keystone-manage trust_flush command
+	TrustFlushArgs string `json:"trustFlushArgs"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="1 * * * *"
+	// TrustFlushSchedule - Schedule to purge expired or soft-deleted trusts from database
+	TrustFlushSchedule string `json:"trustFlushSchedule"`
+
```

Comment:
do we need to be able to control suspend of the job, or is setting a really long schedule the alternative to not run it for the time being?

Reply:
https://kubernetes.io/blog/2021/04/12/introducing-suspended-jobs/

Reply:
I think we can't use that suspend interface because jobs are created by cronjob controller. I checked the current interface of CronJob resource but could not find the way to disable jobs, We probably should remove the CronJob resource in case we want to disable job executuins.

Reply:
I noticed I was looking at something wrong aid CronJob does support suspend. I've updated the change to support it.